### PR TITLE
Create basic calculator

### PR DESCRIPTION
**Pull request by ViktoriiaVolkova**
Create basic calculator, tests and description file for it.

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "singleQuote": true,
```
Comment (4m9fk) [inline .prettierrc.json]: Let's set false here. To keep original formatting

File: tasks/math/calculator/index.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+type operatorRange = '+' | '-' | '/' | '*';
```
Comment (4m9fk) [inline tasks/math/calculator/index.ts]: Usually the types names capitalized

File: tasks/math/calculator/index.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    case '+':
+      return a + c;
+    case '-':
+      return a - c;
+    case '/':
+      return a / c;
+    case '*':
+      return a * c;
+    default:
+      // throw new Error();
```
Comment (4m9fk) [inline tasks/math/calculator/index.ts]: Please remove

File: tasks/math/calculator/index.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      const exhaustiveCheck: never = operator;
+      throw new Error();
```
Comment (4m9fk) [inline tasks/math/calculator/index.ts]: Let's add specific message to the error

File: tasks/math/calculator/index.test.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  it("should return 4", () => {
+    expect(calculator(2, "*", 2)).toBe(4);
+  });
+  it("should return 2", () => {
+    expect(calculator(4, "/", 2)).toBe(2);
+  });
+  it("should return 3", () => {
+    expect(calculator(7, "-", 4)).toBe(3);
+  });
+  expect(calculator).toThrowError(new Error("Calculator does not support the provided operator"));
```
Comment (siarhei-papkou) [inline tasks/math/calculator/index.test.ts]: .

File: tasks/math/calculator/README.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Basic Calculator
+
+Create a function that takes two numbers and a mathematical operator + - / \* and will perform a calculation with the given numbers.
+
+### Examples
+
+> calculator(2, "+", 2) ➞ 4
+
+> calculator(2, "\*", 2) ➞ 4
```
Comment (siarhei-papkou) [inline tasks/math/calculator/README.md]: .

File: tasks/math/calculator/index.test.ts [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    expect(calculator(2, "*", 2)).toBe(4);
+  });
+  it("should return 2", () => {
+    expect(calculator(4, "/", 2)).toBe(2);
+  });
+  it("should return 3", () => {
+    expect(calculator(7, "-", 4)).toBe(3);
+  });
+  it("should return error", () => {
+    expect(calculator).toThrowError(new Error("Calculator does not support the provided operator"));
```
Comment (siarhei-papkou) [inline tasks/math/calculator/index.test.ts]: You should hace call statement to test this case properly
image

**Review by 4m9fk — APPROVED** [review]

**Pull request merged**